ebpf/aya-ebpf/src/programs/xdp.rs

```diff
@@ -38,6 +32,18 @@ impl XdpContext {
     pub fn metadata_end(&self) -> usize {
         self.data()
     }
+
```

Comment:
Not sure why this moved?

Reply:
I put it in the same order as the fields in `ctx`